### PR TITLE
Resolve "Add release step to circle ci config"

### PR DESCRIPTION
**Pull request by lcsrinaldi**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -87,3 +106,29 @@ workflows:
           requires:
             - test
             - lint
+  release:
+    jobs:
+      - checkout:
+          filters:
+            <<: *release_filter
```
Comment (mattii) [inline .circleci/config.yml]: Does it work? I think now it's only running for tagged releases, isn't it?

Comment (lcsrinaldi) [reply .circleci/config.yml]: Yes it works, it's releasing https://circleci.com/gh/picter/prisma/334

Should it work in other cases?

Comment (lcsrinaldi) [reply .circleci/config.yml]: Bear in mind that there are two workflows `default` which runs for every branch and PR and `release` which is only for tags that match the regex.

Comment (mattii) [reply .circleci/config.yml]: Ah ok, sorry. Now I got it 🙂

**Pull request merged**